cloud/deployment/inspect/inspect.go

```diff
@@ -163,6 +164,9 @@ func getDeploymentInfo(sourceDeployment *astro.Deployment, coreDeployment astroc
 		}
 		releaseName = notApplicable
 	}
+	if coreDeployment.WorkloadIdentity != nil {
+		workloadIdentity = *coreDeployment.WorkloadIdentity
+	}
 	return map[string]interface{}{
```

Comment:
can this be changed to `map[string]string`?

Reply:
Doing so, gives an error as createdAt and updatedAt are time based types which do not conforms to string type.

Reply:
yeah my point is we should format those strings here the way we want. can simply do `time.String()` if that's the format we want

Reply:
https://github.com/astronomer/astro-cli/issues/1363